recipes/openssl/1.x.x/conanfile.py

```diff
@@ -433,16 +433,6 @@ def adjust_path(path):
     def _get_env_build(self):
         if not self._env_build:
             self._env_build = AutoToolsBuildEnvironment(self)
-            if self.settings.compiler == "apple-clang":
-                # add flags only if not already specified, avoid breaking Catalyst which needs very special flags
-                flags = " ".join(self._env_build.flags)
-                if "-arch" not in flags:
-                    self._env_build.flags.append("-arch %s" % tools.to_apple_arch(self.settings.arch))
-                if "-isysroot" not in flags:
-                    self._env_build.flags.append("-isysroot %s" % tools.XCRun(self.settings).sdk_path)
```

Comment:
should it be removed?

Reply:
yes, they are already there: https://github.com/conan-io/conan/blob/10e7d5b1e534cc665fa4c403cb560e5de20364d6/conans/client/build/autotools_environment.py#L359